translations/en.json

```diff
@@ -637,6 +637,7 @@
 			"currency-price": "<0>{{currencyKey}}</0> Price",
 			"synthetic-currency-name": "Synthetic {{currencyName}}",
 			"trade-to-currency": "Trade to <0>{{currencyKey}}</0>",
+			"bridge-currency": "Bridge <0>{{currencyKey}}</0>",
 			"buy-currency": "Buy <0>{{currencyKey}}</0>",
```

Comment:
I think you can drop `buy-currency` if it's not used anywhere else.

Reply:
It's being used in https://github.com/Kwenta/kwenta/blob/10f59cd74aff8f556bc5658103bb7e84493a8db2/sections/exchange/FooterCard/NoSynthsCard/NoSynthsCard.tsx#L58